### ISSUE TITLE
[Enhancement] Services configuration workflow improvements

### ISSUE THREAD

**Issue by CubicReg** [centreon/centreon#8373]
Hello,

Here are suggestions for admin profile that would be be convenient in « Configuration > Services > Services by host »:

- Being able to add a column containing the state of each service on their respective lines (ok, warning, critcal, unknow, pending) with a pop-up displaying the output message when mouseovering the state icon.

- Being able to force immediate check in that same view. That way, after modifying a service we could check the result immediately without having to open a « Monitoring > Status Details > Services » tab.

Greetings.

**Comment by hqm199** [comment]
i agree that , in 2104 i can't find way to check the result immediately. who can tell me how to check one service immediately?
